hack/util.sh

```diff
@@ -612,10 +612,11 @@ function util::get_GO_LDFLAGS() {
   # Git information
   GIT_VERSION=$(git describe --tags --dirty)
   GIT_COMMIT_HASH=$(git rev-parse HEAD)
-  GIT_TREESTATE="clean"
-  GIT_DIFF=$(git diff --quiet >/dev/null 2>&1; if [[ $$ = 1 ]]; then echo "1"; fi)
-  if [[ "${GIT_DIFF}" == "1" ]];then
-      GIT_TREESTATE="dirty"
+  # Check if the tree is dirty. defaults to dirty
```

Comment:
What's the meaning of `defaults to dirty`?

Reply:
@RainbowMango 

It‘s copied from https://github.com/kubernetes/kubernetes/blob/master/hack/lib/version.sh#L63

Reply:
> @RainbowMango
> 
> It‘s copied from https://github.com/kubernetes/kubernetes/blob/master/hack/lib/version.sh#L63
> 
> I guess that it refer to the `else` code block

It makes me confused as we don't have default value for `GIT_TREESTATE`.

Reply:
@mrlihanbo It's removed.

Reply:
Thanks.